examples/vite-react/config/jest/setupTestsBeforeEnv.js

```diff
@@ -0,0 +1,6 @@
+import { jestPreviewConfigure } from 'jest-preview';
+
+jestPreviewConfigure({
+  externalCss: ['src/index.css', 'src/global-style.scss'],
+  sassLoadPaths: ['src'], // Root for @use, @import
+});
```

Comment:
@ntt261298 Is there any reason we need to move `jestPreviewConfigure` from `setupFilesAfterEnv` to `setupFiles`?

Reply:
I thought that `setupFilesAfterEnv ` will be run before the transformers, but actually both `setupFilesAfterEnv` and `setupFiles` are run before transformers, so I reverted to use `setupFilesAfterEnv `